_docs_use-cases/use-case-consumption-create-outgoing-request.md

```diff
@@ -58,7 +58,7 @@ The created LocalRequest needs to be manually submitted to the peer, e.g. [by se
 
 ## On Failure
 
-- The request cannot be created if the peer is unknown.
-- The request cannot be created if the request content is malformed. Please [check if the outgoing request can be created](use-case-consumption-check-if-outgoing-request-can-be-created) for more details.
+- The LocalRequest cannot be created if the `peer` is unknown.
+- The LocalRequest cannot be created if the Request specified as `content` is malformed. Please [check if the outgoing request can be created](use-case-consumption-check-if-outgoing-request-can-be-created) for more details.
```

Comment:
Would also leave it here (but set it off as an info box) if it's ok with you, to prevent someone not having read the text above and then being confused. 🦺

Reply:
Yeah I guess there is no harm in repeating it here.